test/nodes/test_whisper.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+import pytest
+
+from haystack.nodes.audio import WhisperTranscriber
+from ..conftest import SAMPLES_PATH
+
+
+@pytest.mark.skipif(os.environ.get("OPENAI_API_KEY", "") == "", reason="OpenAI API key not found")
+def test_whisper_transcribe():
```

Comment:
Shouldn't this test be marked with `@pytest.mark.integration`?

Reply:
Oh right!! Thank you